http/headers/Range.json

```diff
@@ -35,6 +35,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cors_safe": {
+          "__compat": {
+            "description": "CORS-safelisted request header if single <code>bytes</code> range provided",
```

Comment:
Could we make the desc shorter to:
"CORS-safelisted request header with single <code>bytes</code> range"

OR

"CORS-safelisted request header"

OR

"CORS-safelisted <code>Range</code> request header"

Reply:
Thanks a lot:

>Could we make the desc shorter to:
"CORS-safelisted request header with single bytes range

Going with this one as the single `bytes` range is important to keep in, I think.